### ISSUE TITLE
Train Resolution mismatch

### ISSUE THREAD

**Issue by kikixiong** [roboflow/rf-detr#347]
Hello,

I found that the tutorial states that the input resolution [must be divisible by 56.](https://rfdetr.roboflow.com/learn/train/?utm_source=chatgpt.com#fine-tuning) However, when I fine-tune RF-DETR Medium with a resolution that satisfies this (e.g., 1288 = 56 × 23), training crashes with an assertion that the backbone requires divisibility by 32.

<img width="1216" height="100" alt="Image" src="https://github.com/user-attachments/assets/a840a160-a214-409a-844e-0fdbfa94aa5e" />

## Questions

1. **Which constraint is correct for training—56 or 32?**  
   - If both are required (e.g., backbone stride vs. detection head constraints), could you please document that explicitly and provide a concise rule (for example: “choose resolutions divisible by **LCM(32, 56) = 224**”)?

2. **What exact resolution(s) were used to train the official RF-DETR checkpoints** (Small / Medium / Base / Large)?  
   - Single fixed resolution or multi-resolution schedule?  
   - If multi-resolution, what ranges/steps were used?

3. If the backbone prefers sizes divisible by 32 or 56, could you please share the practical trade-offs

4. Does the constraint differ by model size/backbone (e.g., Medium vs. Large)?

**Comment by isaacrob-roboflow** [comment]
Oops! Documentation issue. 

Base and large require 56, nano small and medium require 32. Just has to be divisible by patch size times number of windows, the settings of which are visible in the configs for the models. 

All models are pretrained on multiple resolutions, but the specifics are complex and will be made clear in the upcoming paper. The intention is that the models will support native finetuning at a wide range of resolutions.

**Comment by SlippyDong** [comment]
The documentation still mentions: <html>
<body>

resolution | Sets the input image dimensions. Higher values can improve accuracy but require more memory and can slow training. Must be divisible by 56.

</body>
</html>

This, together with a pending pull request to [support dynamic resolutions](https://github.com/roboflow/rf-detr/pull/79/files), make it inherently difficult to figure out what config to use.

When I checked the cached medium onnx version, which is loaded by the SDK, the environments.json lists:
"PREPROCESSING": "{\"resize\": {\"enabled\": true, \"format\": \"Stretch to\", \"height\": \"576\", \"width\": \"576\"}}",
While an inspection of the model shows:

=== weights.onnx ===
Opset: 17
Nodes: 2135  |  Initializers: 354
Initializer dtype counts: {1: 354}
Total tensor bytes (approx): 108.7 MB
Quantization ops: none
Input shape: 1x3x512x512 | static

This clarified that input resolution must be a multiple of 32 but I'm confused by the stretching to 576x576, which leads me to think that the provided config was for another build as the graph shows that 512x512 is used at runtime.
